proto/hummock.proto

```diff
@@ -75,11 +89,11 @@ message HummockVersionDelta {
   uint64 prev_id = 2;
   // Levels of each compaction group
   map<uint64, LevelDeltas> level_deltas = 3;
-  uint64 max_committed_epoch = 4;
+  uint64 max_committed_epoch = 5;
```

Comment:
We shouldn't change original indexes. Same below.

Reply:
> We shouldn't change original indexes. Same below.

fixed